src/sonic-config-engine/tests/test_j2files.py

```diff
@@ -77,15 +77,6 @@ def test_ipinip(self):
 
         assert filecmp.cmp(sample_output_file, self.output_file)
 
-    def test_everflow(self):
```

Comment:
Why are we removing this test? We still need the minigraph parser to be able to generate mirror sessions from minigraph.

Reply:
Will update test

Reply:
To work with config DB format